pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -53,7 +53,7 @@ in stdenv.mkDerivation rec {
     # Remove buildconfig.html to prevent a dependency on clang etc.
     ./no-buildconfig.patch
   ]
-  ++ lib.optional (lib.versionOlder version "69")
+  ++ lib.optional (lib.versionOlder version "68.3.0")
```

Comment:
Does this mean that 68.3.0 itself shall not have [this patch](https://hg.mozilla.org/integration/autoland/rev/562655fedb9d) applied?

If so, is it nonetheless best practice to leave that lib.optional clause in the derivation, now that the version _is_ at that level?

Reply:
Removed the patch.